src/commands/init.ts

```diff
@@ -0,0 +1,549 @@
+  };
+
+  public async run(): Promise<void> {
+    const { args, flags } = await this.parse(Init);
+
+    try {
+      await this.setGotConfig(flags);
+
+      this.log(
+        chalk.yellow(`This command interactively creates a basic liara.json configuration file.
+It includes only the essential settings; additional configurations must be added manually.
+📚 For more details on each field and its usage, visit: https://docs.liara.ir/paas/liarajson/.
+
+Afterwards, use liara deploy to deploy your app.
+
+🔑 Press ^C at any time to quit.
+`),
+      );
+
+      this.spinner = ora();
+
+      if (flags.y) {
+        try {
+          const dirName = path.basename(process.cwd());
+
+          const platform = detectPlatform(process.cwd());
+          const diskConfig = [
+            {
+              disk: 'media',
+              path: '/uploads/media',
+            },
+          ];
+          const configs = this.setLiaraJsonConfigs(
+            getPort(platform) || 3000,
+            dirName,
+            'iran',
+            platform,
+            supportedVersions(platform)?.defaultVersion,
+            diskConfig,
+          );
+          await this.createLiaraJsonFile(configs);
+          this.log(
+            chalk.yellow(
+              "🚫 This file is just a sample file, don't use it for deployment.",
+            ),
+          );
+
+          this.exit(0);
+        } catch (error) {
+          this.spinner.stop();
+          throw error;
+        }
+      }
+
+      const projects = await this.getPlatformsInfo();
+      const appName = await this.promptProjectName(projects, flags.name);
+      const buildLocation = await this.buildLocationPrompt(
+        flags['build-location'],
+      );
+      const platform = await this.findPlatform(
+        projects,
+        appName,
+        flags.platform,
+      );
+      const port = await this.getAppPort(platform, flags.port, projects);
+      const version = await this.promptPlatformVersion(platform, flags.version);
+      const disks = await this.getAppDisks(appName, projects);
+      const diskConfigs = await this.promptDiskConfig(
+        disks,
+        flags.disk,
+        flags.path,
+      );
+      const cron = await this.promptCron(platform);
+      const healthCheck = await this.promptHealthCheck();
+      const configs = this.setLiaraJsonConfigs(
+        port,
+        appName,
+        buildLocation,
+        platform,
+        version,
+        diskConfigs,
+        healthCheck,
+        cron,
+      );
+
+      await this.createLiaraJsonFile(configs);
+    } catch (error) {
+      this.spinner.stop();
+      throw error;
+    }
+  }
+
+  async getPlatformsInfo(): Promise<IProject[]> {
+    try {
+      this.spinner.start();
+
+      const { projects } =
+        await this.got('v1/projects').json<IGetProjectsResponse>();
+
+      this.spinner.stop();
+      return projects as IProject[];
+    } catch (error) {
+      if (error.response && error.response.statusCode === 401) {
```

Comment:
Do we need this check? We have a global error handler for 401 error code.

Reply:
That's correct! However, as you can see in the error message below:  
```
Authentication failed.  
Please log in using the 'liara login' command.

If you are using an API token for authentication, please consider updating your API token.  
You can still create a sample 'liara.json' file using the 'liara init -y' command.
```
I'm showing an additional message so the user know that he can creates a liara.json with -y flag.